addon/data/message-bridge-readonly.js

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
As a nitpick, I'd say this shouldn't be named "message-bridge" - because it's not actually doing the addon port / content event bridging. Maybe something more like `set-installed-flag`

Also, to reduce redundant work, if we're setting the flag here we should probably take that side effect out of `message-bridge.js` and just load this same PageMod on the Test Pilot site, too